management-proxy/session-manager/routes/system.js

```diff
@@ -58,8 +58,8 @@ module.exports = function (router) {
           if (response.statusCode == HttpStatus.OK && result.namespace) {
             var get_image_id_for_key = result.keys.reduce(function (obj, key) {
               // only compare images belonging to the current channel
-              // this takes care of images like ibuildthecloud/..., too
-              if (key.indexOf(channel) >= 0) {
+              // exclude buildstep as it's currently only available w/ tag :latest.
+              if (key.match(channel + "$") == channel && key.indexOf('buildstep:') === -1) {
```

Comment:
not sure about ``key.match(channel + "$") == channel`` since ``match`` usually returns an array or ``null`` but not a string